source/_integrations/roon.markdown

```diff
@@ -28,3 +28,16 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
 3. Home Assistant will then contact your Roon Core and ask to be authorized. You will need to enable this extension in the Room Application. Go to **Settings** and then **Extensions**, there you will see an entry for Home Assistant with a button next to it. Click **Enable**.
 4. Roon core will then provide Home Assistant with the details of your media players.
 5. In Home Assistant you can then pick an area for each of your music players, and add them to Home Assistant.
+
+#### Service `media_player.play_media`
```

Comment:
This is documented in the media player, and not a service specifically for Roon. It should not be documented here.

Reply:
@frenck the media player documentation says “A media identifier. The format of this is integration dependent. For example, you can provide URLs to Sonos and Cast but only a playlist ID to iTunes.”

This is particularly true here because of the limitations of this platform.

All I really need to include are my notes about what the two parameters need to be for this player. Could easily be a note or something, but otherwise someone will need to read the code.

So if you can point me to a similar case happy to re-work.to fit the right  pattern.

Reply:
The right pattern is not to document services that are not part of this integration.
It is a media_player service, not a roon service.

Reply:
@frenck Open to do this in what ever way you think best, please make a suggestion as to what to do, rather than what not to do :-)

The plex player has done pretty much as I have - and redocumented `media_player/play`
https://www.home-assistant.io/integrations/plex/

As does DirectTV
https://www.home-assistant.io/integrations/directv/

The Samsung player has a useage section
https://www.home-assistant.io/integrations/samsungtv/

Reply:
ok, lets keep it for now. I still think we should avoid this type of documentation in general.